flowzone.yml

```
@@ -4,13 +4,20 @@
     uses: actions/download-artifact@v3
     with:
       name: source-${{ github.event.pull_request.head.sha || github.event.head_commit.id }}
-      path: /tmp
+      path: ${{ runner.temp }}
 
   - &extractSourceArtifact
     name: Extract source artifact
+    if: runner.os != 'Windows'
     working-directory: .
-    shell: bash
-    run: tar -xvf /tmp/source.tgz
+    run: tar -xvf ${{ runner.temp }}/source.tgz
+
+  - &extractSourceArtifactWindows
+    name: Extract source artifact
+    if: runner.os == 'Windows'
+    shell: powershell
+    working-directory: .
+    run: tar -xvf ${{ runner.temp }}\source.tgz
```

It should also with fine with `/` on windows, so this should be able to match the linux version
```suggestion
    run: tar -xvf ${{ runner.temp }}/source.tgz
```

---

It works fine in powershell (6+7 at least), although it can cause issues in cmd - `/` is used for args so eg `dir ./foo` would give `Invalid switch - "foo".` but `dir "./foo"` works

---

@Page- here's the issue: https://github.com/balena-io/etcher/actions/runs/3386762422/jobs/5631091011#step:3:10